### PR TITLE
OSDOCS-5196: Updated 7.0 version in VSphere docs

### PR DESCRIPTION
**Pull request by dfitzmau**
Issue:
[OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197)

Version(s):
4.13

Link to docs preview:
* [Preparing to install on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/preparing-to-install-on-vsphere.html)
* [Installing a cluster on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by dfitzmau** [comment]
Hi @rvanderp3 .

I updated the version number as per the docs update request in [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197). Please let me know if these updates capture what you requested. I added preview links at the start of this PR that each show the changes. 

I see the following note in the docs that might also require update to the version numbers? However, `7.0 update 0` sounds a bit off for some reason (as in it's not an update). Please let me know your thoughts. 

Screenshot from 2023-02-14 13-12-17

**Comment by rvanderp3** [comment]
> Issue: [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197)
> 
> Version(s): 4.3
> 
> Link to docs preview:
> 
> * [Preparing to install on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/preparing-to-install-on-vsphere.html)
> * [Installing a cluster on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html)
> 
> QE review:
> 
> * [ ]  QE has approved this change.

I think you mean 4.13 here

**Comment by rvanderp3** [comment]
> Hi @rvanderp3 .
> 
> I updated the version number as per the docs update request in [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197). Please let me know if these updates capture what you requested. I added preview links at the start of this PR that each show the changes.
> 
> I see the following note in the docs that might also require update to the version numbers? However, `7.0 update 0` sounds a bit off for some reason (as in it's not an update). Please let me know your thoughts.
> 
> Screenshot from 2023-02-14 13-12-17

@jcpowermac 7.0 and 7.0u1 aren't just deprecated are they?  You must be running 7.0u2 or later, right?

**Comment by rvanderp3** [comment]
> > Hi @rvanderp3 .
> > I updated the version number as per the docs update request in [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197). Please let me know if these updates capture what you requested. I added preview links at the start of this PR that each show the changes.
> > I see the following note in the docs that might also require update to the version numbers? However, `7.0 update 0` sounds a bit off for some reason (as in it's not an update). Please let me know your thoughts.
> > Screenshot from 2023-02-14 13-12-17
> 
> @jcpowermac 7.0 and 7.0u1 aren't just deprecated are they? You must be running 7.0u2 or later, right?

though i guess i'm not clear on which version this PR is targeting, for 4.12, deprecated may be the right language.

**Comment by jcpowermac** [comment]
I think we should have PM comment @racedo

**Comment by dfitzmau** [comment]
> > Issue: [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197)
> > Version(s): 4.3
> > Link to docs preview:
> > 
> > * [Preparing to install on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/preparing-to-install-on-vsphere.html)
> > * [Installing a cluster on vSphere](https://55872--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html)
> > 
> > QE review:
> > 
> > * [ ]   QE has approved this change.
> 
> I think you mean 4.13 here

Yes. Thanks. Well spotted!

**Comment by dfitzmau** [comment]
> > > Hi @rvanderp3 .
> > > I updated the version number as per the docs update request in [OSDOCS-5196](https://issues.redhat.com/browse/OSDOCS-5197). Please let me know if these updates capture what you requested. I added preview links at the start of this PR that each show the changes.
> > > I see the following note in the docs that might also require update to the version numbers? However, `7.0 update 0` sounds a bit off for some reason (as in it's not an update). Please let me know your thoughts.
> > > Screenshot from 2023-02-14 13-12-17
> > 
> > 
> > @jcpowermac 7.0 and 7.0u1 aren't just deprecated are they? You must be running 7.0u2 or later, right?
> 
> though i guess i'm not clear on which version this PR is targeting, for 4.12, deprecated may be the right language.

Hi @rvanderp3 . This PR targets the 4.13 release, so deprecated might still be OK? 

Version numbers should state 7.0 update 0 and 7.0 update 1? If not, why the change for 7.0 update 2? We usually try to maintain consistency among terms, versions, and so on; this avoids distracting content that can cause a doc's user to lose their focus.

**Comment by racedo** [comment]
This is to ensure that *OpenShift 4.13* on vSphere is limited to vSphere 7.0 Update 2 or later for installations and upgrades, because OCP 4.13 will have CSI migration enabled and vSphere 7.0 Update 2 is the minimum version that meets this requirement of migration. 

Earlier vSphere versions are not supported in OCP 4.13.

Upgrades to OpenShift 4.13 will be blocked if vSphere 7.0 Update 1 or earlier is used. vSphere 7.0 Update 2 or later is the minimum version required for upgrade.

Also, vSphere 7.0 update 0 isn't referenced like that anywhere, it's simply vSphere 7.0. Using "Update"  (note vmware users uppercase  U) starts in "Update 1".

**Comment by dfitzmau** [comment]
Thanks, @racedo , for the comments. I updated the version number references according to your feedback. 

I can see your point about the deprecated notice. I'm also think the notice does not apply well to the strict 7.0 Update 2 or later versions for 4.13. The notice should mention that 7.0 and 7.0 Update 1 are not compatible with installing OCP 4.13 on vSphere?

**Comment by dfitzmau** [comment]
I also see this paragraph in the pre-existing text:

> You must install the {product-title} cluster on a VMware vSphere version 7.0 Update 2 or later instance that meets the requirements for the components that you use.

Cluster is mentioned here also, but maybe it should state the high-level OCP entity? This would help clarify your last comment along with an update to the notice (remove the cluster reference)?

**Comment by rvanderp3** [comment]
LGTM
cc: @jcpowermac

File: modules/installation-vsphere-infrastructure.adoc [hunk 1]
```diff
@@ -45,39 +45,43 @@ endif::[]
...
-|vSphere ESXi hosts | 7.0.2 or later
-|vCenter host   | 7.0.2 or later
+|vSphere ESXi hosts | 7.0 Update 2 or later
+|vCenter host   | 7.0 Update 2 or later
 |===
 
 [IMPORTANT]
 ====
-Installing a cluster on VMware vSphere versions 7.0.0 and 7.0.1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported. Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later. To update the hardware version for your vSphere virtual machines, see the "Updating hardware on nodes running in vSphere" article in the _Updating clusters_ section.
+Installing a cluster on VMware vSphere versions 7.0 and 7.0 Update 1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported. Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later. To update the hardware version for your vSphere virtual machines, see the "Updating hardware on nodes running in vSphere" article in the _Updating clusters_ section.
```
Comment (duanwei33) [inline modules/installation-vsphere-infrastructure.adoc:66]: > Installing a cluster on VMware vSphere versions 7.0 and 7.0 Update 1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported.

I understand we don't support installation on 7.0.1 in 4.13, right?



> Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later

And we should mention 4.13 here.

cc @sgaoshang

Comment (dfitzmau) [reply modules/installation-vsphere-infrastructure.adoc]: Thanks, @duanwei33 . I updated the content to state 4.13.

I'll await @sgaoshang response on dropping the 7.0 and 7.0 Update 1 sentence.

Comment (sgaoshang) [reply modules/installation-vsphere-infrastructure.adoc]: I'm also prefer we drop the 7.0 and 7.0 Update 1 sentence, thanks.

Comment (jcpowermac) [reply modules/installation-vsphere-infrastructure.adoc]: the 4.13 installer will not allow below vSphere 7 U2 components...that includes *all* esxi hosts in the defined clusters and vcenter.

Comment (dfitzmau) [reply modules/installation-vsphere-infrastructure.adoc]: Thank you, all. I removed this admonition from the text.

File: modules/installation-vsphere-infrastructure.adoc [hunk 2]
```diff
@@ -45,39 +45,38 @@ endif::[]
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 7 instance that meets the requirements for the components that you use.
+You must install the {product-title} cluster on a VMware vSphere version 7.0 Update 2 or later instance that meets the requirements for the components that you use.
 
+[NOTE]
+====
+Version 4.13 of {product-title} supports VMware vSphere version 8.0.
```
Comment (mburke5678) [inline modules/installation-vsphere-infrastructure.adoc:52]: s/Version 4.13 of {product-title}/ {product-title} version 4.13

**Comment by dfitzmau** [comment]
Thanks, @mburke5678 . I updated the content based on your suggestion. Please let me know your thoughts.

**Comment by ousleyp** [comment]
LGTM. Merging shortly!

**Pull request merged**